plugin/vcs/vcs.go

```diff
@@ -24,6 +24,17 @@ func (e Type) String() string {
 	return "UNKNOWN"
 }
 
+// OAuthToken is the API message for OAuthToken.
+type OAuthToken struct {
+	AccessToken  string `json:"access_token" `
+	RefreshToken string `json:"refresh_token"`
+	ExpiresIn    int64  `json:"expires_in"`
+	CreatedAt    int64  `json:"created_at"`
+	// ExpiresTs is a derivative from ExpresIn and CreatedAt.
+	// ExpiresTs = ExpiresIn == 0 ? 0 : CreatedAt + ExpiresIn
+	ExpiresTs int64 `json:"expires_ts"`
+}
```

Comment:
It's derived instead of getting from gitlab api response, so we do not need to have the json annotation?

Reply:
we still need this because the frontend need it. (since we moved the derivative logic to the backend)